draft-birkholz-scitt-architecture.md

```diff
@@ -162,56 +162,56 @@ Claim:
 
 {: #mybody}
 
-# Definition of Transparency
+# Definition of Verifiability
 
-In this document, we use a definition of transparency built over abstract notions of Ledgers and Receipts. Existing transparency systems such as Certificate Transparency are instances of this definition.
+In this document, we use a definition of verifiability built over abstract notions of Receipts and Ledgers. Existing verifiability systems such as Certificate Transparency are instances of this definition, though simpler systems such as timestamping services also satisfy the definition.
 
-A Claim is an identifiable and non-repudiable Statement made by an Issuer. The Issuer selects additional metadata and attaches a proof of endorsement (in most cases, a signature) using the identity key of the Issuer that binds the Statement and its metadata. Claims can be made transparent by attaching a proof of Registration by a TS, in the form of a Receipt that countersigns the Claim and witnesses its inclusion in the Ledger of a TS. By extension, we may say an Artifact (e.g. a firmware binary) is transparent if it comes with one or more Transparent Claims from its author or owner, though the context should make it clear what type of Claim is expected for a given Artifact.
+A Claim is an identifiable and non-repudiable Statement made by an Issuer. The Issuer selects additional metadata and attaches a proof of endorsement (in most cases, a signature) using the identity key of the Issuer that binds the Statement and its metadata. Claims can be made verifiable by attaching a proof of Registration by a RS, in the form of a Receipt that countersigns the Claim and witnesses its processing by the RS. By extension, we may say an Artifact (e.g. a firmware binary) is verifiable if it comes with one or more Verifiable Claims from its author or owner, though the context should make it clear what type of Claim is expected for a given Artifact.
 
-Transparency does not prevent dishonest or compromised Issuers, but it holds them accountable: any Artifact that may be used to target a particular user that checks for Receipts must have been recorded in the tamper-proof Ledger, and will be subject to scrutiny and auditing by other parties.
+Verifiability can be implemented in a variety of ways depending on the desired properties. For example, a verifiability of receipts from a RS might be implemented with a Ledger that provides a consistent, append-only, publicly available record of entries. Implementations of RS may protect their Ledger using a combination of trusted hardware, replication and consensus protocols, and cryptographic evidence. A Receipt is an offline, universally-verifiable proof that an entry is recorded in the edger. Receipts do not expire, but it is possible to append new entries that subsume older entries.
```

Comment:
The definition of ledger is no longer valid when the ledger is optional. A receipt is now a witness that a claim has been registered on SCITT service, with optional universally verifiable proof that the claim has been appended to the ledger.